### PR TITLE
Leaflet functions: add_bc_home_button and set_bc_view

### PR DESCRIPTION
**Pull request by ateucher**
```r
library(leaflet)
library(envreportutils)

leaflet() %>% 
  addProviderTiles(providers$OpenStreetMap) %>% 
  set_bc_view() %>% 
  add_bc_home_button()
```

### CODE REVIEW

File: data-raw/bc_button_svg.R [hunk 1]
```diff
@@ -0,0 +1,17 @@
+library(sf)
+library(bcmaps)
+library(rmapshaper)
+library(svglite)
+library(base64enc)
+
+bc_simp <- bc_bound() %>% 
+  st_geometry() %>%
```
Comment (boshek) [inline data-raw/bc_button_svg.R:8]: What do you think about projecting BC to the same projection as Leaflet? 
` sf::st_transform(3857) %>% ` would do it.

Comment (ateucher) [reply data-raw/bc_button_svg.R:8]: image

Comment (ateucher) [reply data-raw/bc_button_svg.R:8]: That's how it looks with matching the Leaflet projection. Thoughts @stephhazlitt ?

Comment (stephhazlitt) [reply data-raw/bc_button_svg.R:8]: I think in theory the most consistent approach would be to have the icon in the same projection as the leaflet map. However, it is harder for me to recognize British Columbia (harder to make out Haida Gwaii, V.I.). 🤷

Comment (boshek) [reply data-raw/bc_button_svg.R:8]: Agreed - that actually does not look as good.

Comment (ateucher) [reply data-raw/bc_button_svg.R:8]: I think I like the 'normal' projection better as well.

**Review by stephhazlitt — APPROVED** [review]
👍 Nice work @ateucher

**Pull request merged**